x-pack/plugins/ml/server/lib/alerts/jobs_health_service.ts

```diff
@@ -99,17 +131,36 @@ export function jobsHealthServiceProvider(
             const jobState =
               jobsStats.find((jobStats) => jobStats.job_id === jobId)?.state ?? 'failed';
             return {
-              ...datafeedStats,
+              datafeed_id: datafeedStats.datafeed_id,
+              datafeed_state: datafeedStats.state,
               job_id: jobId,
               job_state: jobState,
             };
           })
           .filter((datafeedStat) => {
             // Find opened jobs with not started datafeeds
-            return datafeedStat.job_state === 'opened' && datafeedStat.state !== 'started';
+            return datafeedStat.job_state === 'opened' && datafeedStat.datafeed_state !== 'started';
           });
       }
     },
+    /**
+     * Gets jobs that reached soft or hard model memory limits.
+     * @param jobIds
+     */
+    async getMmlReport(jobIds: string[]): Promise<MmlTestResponse[]> {
+      const jobsStats = await getJobStats(jobIds);
+
+      return jobsStats
+        .filter((j) => j.model_size_stats.memory_status !== 'ok')
```

Comment:
I think it's worth adding a constants file with an `enum` for the possible `memory_status` values - `ok`, `hard_limit` and `soft_limit`.

Reply:
it comes from the elasticseach client with a union type, so it's safe to use this way